src/client/builtin_scene/web_content_renderer.cpp

```diff
@@ -1074,6 +1089,52 @@ namespace builtin_scene::web_renderer
     return fragment->contentWidth();
   }
 
+  void RenderTextSystem::generateSDFTextureForText(ecs::EntityId entity, WebContent &content, const std::string &text)
```

Comment:
This function should write SDF texture in place, as for SDF texture, it only uses the alpha channel, other channels preserve zero.